test/utils/shippable/integration.sh

```diff
@@ -67,8 +71,12 @@ if [ "${controller_shared_dir}" ]; then
     cp -a "${SHIPPABLE_BUILD_DIR}" "${controller_shared_dir}"
 fi
 
+# Enabling colored output in Shippable console
+color_settings="--env ANSIBLE_FORCE_COLOR=$force_color"
+
 httptester_id=$(docker run -d "${http_image}")
 container_id=$(docker run -d \
+    ${color_settings} \
```

Comment:
Inline ```--env ANSIBLE_FORCE_COLOR=$force_color``` here. Doing so avoids an extra variable and allows this script to pass shellcheck.